config/rule.go

```diff
@@ -42,6 +42,42 @@ type Rule struct {
 	// Allowlist allows a rule to be ignored for specific
 	// regexes, paths, and/or commits
 	Allowlist Allowlist
+
+	// Report indicates whether the rule should be scanned/reported.
+	// This defaults to 'true' but can be set to 'false' for multi-part secret verification.
+	Report bool
+	Verify Verify
+}
+
+type Verify struct {
+	HTTPVerb             string
+	URL                  string
+	Headers              map[string]string
+	ExpectedStatus       []string
```

Comment:
Should this be `[]int`?

Reply:
Initially wrote this as a string to support "2xx", "30x" for example. `[]int` might be better considering the return codes should be known.

Reply:
> might be better considering the return codes should be known.

Yeah. I see the appeal of "2xx" but I think it's too error prone in practice.